pkg/kubectl/cmd/config/get_contexts.go

```diff
@@ -89,7 +79,13 @@ func (o *GetContextsOptions) Complete(cmd *cobra.Command, args []string, out io.
 	o.contextNames = args
 	o.out = out
 	o.nameOnly = false
-	if cmdutil.GetFlagString(cmd, "output") == "name" {
+	output := cmdutil.GetFlagString(cmd, "output")
+	supportedOutputTypes := sets.NewString("", "name")
```

Comment:
Better set a meaningful default value instead of an empty string.

Reply:
use ```""``` as default output better than we have a output type ```table```? we need not introduce this for user